pylint/__pkginfo__.py

```diff
@@ -9,7 +9,7 @@
 
 from __future__ import annotations
 
-__version__ = "2.16.0b0"
+__version__ = "2.16.0-b1"
```

Comment:
```suggestion
__version__ = "2.16.0b1"
```

Reply:
Yes sorry, the tbump script is not working properly with ``0b1`` and I forgot to replace.